README.md

```diff
@@ -164,4 +192,4 @@ explain why.
 
 ## A2.7
 
-«replace this with your answer»
+The simplest Chomsky grammar level for this language is Chomsky Type-2 because in Type-3, the right side of a production is always either a terminal, or a nonterminal followed by a terminal, but in this language the definitions of a sentence contain multiple nonterminals.
```

Comment:
# -3

But you've already shown it can be implemented using an FSM, so it must be type 3...